index.js

```diff
@@ -12,48 +12,53 @@ module.exports.decodeBox2 = function(box2) {
 // assumes msg has already been validated
 // returns a classic compatible json object
 module.exports.decode = function(bmsg) {
-  const decoded = bencode.decode(bmsg)
+  const [payload, signature] = bencode.decode(bmsg)
+  const [author, sequence, previous, timestamp, contentSection] = payload
 
   const result = {
-    previous: bfe.decode(decoded[0][2]),
-    author: bfe.decode(decoded[0][0]),
-    sequence: decoded[0][1],
-    timestamp: decoded[0][3],
-    signature: bfe.decode(decoded[1])
+    previous: bfe.decode(previous),
+    author: bfe.decode(author),
+    sequence: bfe.decode(sequence),
```

Comment:
`sequence` wasn't being decoded through `ssb-bfe` before, because it's a number, and in BFE it's just a passthrough, but `ssb-bendy-butt` shouldn't know about that, that's BFE concerns which we shouldn't prematurely optimize in this module.